client/src/components/DataHome/DataLog/DataLog.jsx

```diff
@@ -198,20 +198,6 @@ function DataLog({ handleSignInOpen, handleCreateAccOpen, isLoggedIn, id }) {
               onChange={(e) => onDrainColorChange(e)}
               value={drainColor}
             />
-            <h3>Drain output photo</h3>
```

Comment:
From this change it looks like you're removing the ability for patients to upload picture. Is that an intended change?

Reply:
Yes I decided that I will work on this part after I finish the provider view and graph representation of data.